lib/swift_utils.py

```diff
@@ -574,6 +574,21 @@ def set_min_part_hours(path, value):
         raise SwiftProxyCharmException(msg)
 
 
+def get_replicas(ring):
+    builder = _load_builder(ring)
+    return builder.replicas
+
+
+def set_replicas(path, value):
+    cmd = ['swift-ring-builder', path, 'set_replicas', str(value)]
+    p = subprocess.Popen(cmd)
+    p.communicate()
+    rc = p.returncode
+    if rc != 0:
+        msg = ("Failed to set_replicas=%s on %s" % (value, path))
```

Comment:
Consider using `.format` here instead of `%` for consistency and version (python 2/3) compatibility.